trax/data/tf_inputs.py

```diff
@@ -1090,30 +1095,136 @@ def download_and_prepare(dataset_name, data_dir):
   return data_dir
 
 
-def bert_single_sentence_preprocess(batch):
-  for sent1, label in batch:
-    value_vector = np.concatenate(([101], sent1, [102]))
-    segment_embs = np.zeros(sent1.shape[0] + 2, dtype=np.int32)
-    yield value_vector, segment_embs, segment_embs, label, np.int64(1)
+def BertSingleSentenceInputs(batch, labeled=True, CLS_id=101, SEP_id=102):
+  """Prepares inputs for BERT models by adding [SEP] and [CLS] tokens and creating segment embeddings."""
+  if labeled:
+    for sent1, label in batch:
+      value_vector = np.concatenate(([CLS_id], sent1, [SEP_id]))
+      segment_embs = np.zeros(sent1.shape[0] + 2, dtype=np.int32)
+      yield value_vector, segment_embs, segment_embs, label, np.int64(1)
+  else:
+    for sent1, *_ in batch: # row is a tuple with 1 element
```

Comment:
@piotrekp1 - thanks for the comment (and teaching me something new) but this syntax is a quite confusing to me, can we be explicit here?

Something like:
```
for row in batch:
  # row is a tuple with 1 element
  sent1 = row[0]
```

Or perhaps -- does this even work?
```
for (sent1,) in batch: # row is a tuple with 1 element
```

Reply:
Actually (sent1, ) worked! Thanks, now you too taught me something about this syntax :)